vividus-plugin-web-app/src/main/java/org/vividus/selenium/WebDriverType.java

```diff
@@ -81,6 +81,7 @@ public void prepareCapabilities(DesiredCapabilities desiredCapabilities)
             // Workaround for IExplore:
             // https://stackoverflow.com/questions/50287435/actions-movetoelement-not-working-on-ie-11
             desiredCapabilities.merge(new InternetExplorerOptions().requireWindowFocus());
+            disableAcceptInsecureCertificates(desiredCapabilities);
```

Comment:
can you please describe in few words why its needed

Reply:
what about just changing this code:
https://github.com/vividus-framework/vividus/blob/master/vividus-extension-selenium/src/main/java/org/vividus/selenium/AbstractVividusWebDriverFactory.java#L68 ?
can we check if IE || Safari and then skip adding of this capability?

the problem with current approach is that users may set the capability and expect the changes in browser behavior, but we just silently remove this capability